db/models/gallery_table.js

```diff
@@ -0,0 +1,6 @@
+const bookshelf = require('./bookshelf');
+
+const Gallery = bookshelf.Model.extend({
+  tableName: 'gallery'
+});
+module.exports = Gallery;
```

Comment:
Refactor your Models to use the "class" style of setting up.

`class Gallery extends bookshelf.Model {`

Reply:
The model file name should be `Gallery.js`